src/ResourceManager/Insights/Commands.Insights/Autoscale/NewAzureRmAutoscaleRuleCommand.cs

```diff
@@ -94,12 +94,6 @@ public class NewAzureRmAutoscaleRuleCommand : AzureRMCmdlet
         public ScaleDirection ScaleActionDirection { get; set; }
 
         /// <summary>
-        /// Gets or sets the scale action type
-        /// </summary>
-        [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, HelpMessage = "The scale action type for the setting")]
-        public ScaleType ScaleActionScaleType { get; set; }
```

Comment:
@gucalder this is a breaking change; you are removing a **mandatory** parameter from this cmdlet, so any script that uses this cmdlet will break